test/loader_spec.js

```diff
@@ -28,7 +28,8 @@ describe('On spin up', function () {
     assert.equal(db.queryFiles.length, 7);
   });
   it('loads up functions', function () {
-    assert.equal(db.functions.length, 20);
+    // newer versions of postgres include more than 20 functions
+    assert.ok(db.functions.length >= 20);
```

Comment:
Rather than switch these tests based on version, I'm just using a `>=` check to avoid getting into the specifics of how many functions each version ships with.